guides/upgrading/index.md

```diff
@@ -1,39 +1,27 @@
 
-## Details about recent releases
+## Détail des _releases_ récentes
 
-You can learn about what changed in every release by reading the 
-[official Ember blog](https://blog.emberjs.com).
+Vous apprendrez ce qui a changé dans chaque _release_ (version livrée) en lisant [le blog officiel d'Ember](https://blog.emberjs.com).
 
-## Routine minor version maintenance upgrades
+## Mises à jour de routine pour maintenir les versions mineures
 
-Let's say you are upgrading an app from `v3.4` to `v3.8`.
-Although you heard there are some new features, your main goal is to keep up with security updates.
-To do this kind of version upgrade, follow the instructions [here in the CLI Guides](https://cli.emberjs.com/release/basic-use/upgrading/).
-That process will make sure that any peer dependencies of the `ember.js` core codebase get upgraded too.
-Automated tools will help you make the right changes to `package.json` and other files.
-The point of a minor version bump is that you shouldn't need to change anything in your app when you upgrade - it should all keep working, whether or not you choose to adopt new syntaxes and features.
+Mettons que vous voulez mettre à jour une app de `v3.4` à `v3.8`. Bien que vous ayez entendu parler de nouvelles fonctionnalités, votre objectif principal est de suivre les mises à jour de sécurité. Pour faire ce type de mise à jour, suivez les instructions [ici, dans les Guides CLI](https://cli.emberjs.com/release/basic-use/upgrading/). Ce processus vous assurera que toutes les dépendances de la base de code `ember.js` sont également mises à jour. Des outils automatisés vous aideront à faire les bons changements dans le `package.json` et d'autres fichiers. L'intérêt d'un saut de version mineure est que vous ne devriez pas avoir besoin de changer quoi que ce soit dans votre application lors de la mise à jour&nbsp;; tout devrait continuer à fonctionner, que vous choisissiez ou non d'adopter de nouvelles syntaxes et fonctionnalités.
 
-## Using new features
+## Utiliser de nouvelles fonctionnalités
 
-Once you have upgraded an app's version, some new features may be available out of the box.
-On the other hand, some features will require that you enable them specifically in your app's configuration, since they may change the app's default behavior.
+Une fois que vous avez mis à jour la version d'une app, de nouvelles fonctionnalités peuvent parfois être disponibles d'emblée. En revanche, d'autres fonctionnalités nécessitent que vous les activiez spécifiquement dans la configuration de votre app, car elles peuvent modifier son comportement par défaut.
```

Comment:
Personally I would write it this way to insist on the fact these features may or may not change the behaviour.
I found the original sentence more like "they can and they will".

```suggestion
Une fois que vous avez mis à jour la version d'une app, de nouvelles fonctionnalités prêtes à l'emploi peuvent parfois être disponibles. En revanche, d'autres fonctionnalités nécessitent que vous les activiez spécifiquement dans la configuration de votre app, car il est possible qu'elles puissent modifier le comportement par défaut de votre app.
```

Nitpicking: I would keep the same line breaks at it makes the reading easier, less compact.